x-pack/plugins/alerting/server/connector_adapters/connector_adapter_registry.ts

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
I will delegate this question to @mikecote 🙂. I remember we discussed it.

Reply:
It can definitely be odd at first glance, but when looking at the structure of a connector adapter (see `connector_adapters/types.ts`), it all relates to alerting plugin types that the actions plugin doesn't have access to. It keeps the actions plugin unaware of alerting (with a few exceptions :shh:).